xwiki/templates/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+      ingress.kubernetes.io/rewrite-target: /
```

Comment:
There's no annotations section in 'values.yaml'. You should probably create one there,  and move the rewrite annotations also there.